woocommerce-gateway-ebanx/templates/baloto/payment-failed.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Baloto - Payment instructions.
+ *
+ * @author  EBANX.com
+ * @package WooCommerce_EBANX/Templates
+ * @version 2.0.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+?>
+
+<div class="woocommerce-message">
+	<span><?php _e('Payment successfully made using Baloto.', 'woocommerce-gateway-ebanx' ); ?></span>
```

Comment:
That is really wrong :confounded: I'll fix it.
But look at the template files from safetypay and servipag. I followed them, should we correct them too?